pkg/restapi/vc/operation/operations.go

```diff
@@ -642,3 +674,18 @@ func (o *Operation) parseAndVerifyVC(vcBytes []byte) (*verifiable.Credential, er
 
 	return vc, nil
 }
+
+func (o *Operation) parseAndVerifyVP(vpBytes []byte) (*verifiable.Presentation, error) {
+	vp, err := verifiable.NewPresentation(
```

Comment:
you can return here

Reply:
due to  https://github.com/trustbloc/edge-service/pull/99#discussion_r388032493 comment, no longer its applied here.